src/main/java/com/hyperwallet/clientsdk/util/HyperwalletApiClient.java

```diff
@@ -69,6 +69,16 @@ public HyperwalletApiClient(final String username, final String password, final
         }
     }
 
+    public <T> T post(final String url, final Class<T> type) {
```

Comment:
I don't think we need a overload of this method only because of there is no body in the new call. 
Creating this new method we are saying to clients that body is optional and actually it is needed even if EMPTY.